Shared/src/commonMain/kotlin/com/conradkramer/wallet/EthereumBalanceUpdater.kt

```diff
@@ -0,0 +1,67 @@
+package com.conradkramer.wallet
+
+import com.conradkramer.wallet.ethereum.Address
+import com.conradkramer.wallet.ethereum.Data
+import com.conradkramer.wallet.ethereum.Quantity
+import com.conradkramer.wallet.ethereum.RpcClient
+import com.conradkramer.wallet.ethereum.Token
+import com.conradkramer.wallet.ethereum.Transaction
+import com.conradkramer.wallet.ethereum.requests.Call
+import com.conradkramer.wallet.ethereum.requests.GetBalance
+import com.conradkramer.wallet.platform.Keccak256Digest
+import io.ktor.utils.io.core.toByteArray
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+import kotlin.coroutines.EmptyCoroutineContext
+
+internal interface BalanceUpdater {
+    fun update(address: Address)
+}
```

Comment:
If a client wanted to wait for the update to be finished, they would be unable to with this interface. I'd suggest maybe doing this to get the best of both worlds:
```suggestion
internal interface BalanceUpdater {
    val scope: CoroutineScope
    suspend fun update(address: Address)
    
    fun update(address: Address) {
        scope.launch {
             update(address)
        }
    }
}
```